proto/protovalidate/buf/validate/validate.proto

```diff
@@ -2711,11 +2718,18 @@ message StringRules {
     //   string value = 1 [(buf.validate.field).string.hostname = true];
     // }
     // ```
-    bool hostname = 13 [(priv.field).cel = {
-      id: "string.hostname",
-      message: "value must be a valid hostname",
-      expression: "this.isHostname()",
-    }];
+    bool hostname = 13 [
+      (priv.field).cel = {
+        id: "string.hostname",
+        message: "value must be a valid hostname",
+        expression: "this.isHostname() || this == ''",
+      },
+      (priv.field).cel = {
+        id: "string.hostname_empty",
+        message: "an empty value is not a valid hostname",
```

Comment:
This is more a statement, it makes me ask the question "...ok, was my value empty?" I'd say something like "value was empty, and empty is not a valid X"

Reply:
updated in [23bed22](https://github.com/bufbuild/protovalidate/pull/133/commits/23bed2208087c8a81fb9be87f7c1300f5a03ff07), which now reads "value is empty, which is not a valid X"